.travis.yml

```diff
@@ -8,15 +9,11 @@ branches:
 matrix:
   fast_finish: true
   include:
-    - python: 2.7
-      script: python setup.py flake8 && python setup.py test
-    - python: 3.3
-      script: python setup.py test
-    - python: 3.4
-      script: python setup.py test
     - python: 3.5
-      script: python setup.py test
-    - python: pypy
+      script: python setup.py flake8 && python setup.py test
+    - python: 3.6
+      script: python setup.py flake8 && python setup.py test
+    - python: 3.7
       script: python setup.py flake8 && python setup.py test
     - python: pypy3
-      script: pip install setuptools_scm && python setup.py test 
+      script: python setup.py flake8 && python setup.py test
```

Comment:
Travis says:
> $ pip install setuptools_scm && python setup.py test
> Requirement already satisfied: setuptools_scm in /home/travis/virtualenv/pypy3.6-7.1.1/site-packages (3.3.3)

Thus, there's no reason to specify `pip install setuptools_scm` at here.